src/modules/legal/assets/templates/legal-notice.template.md

```
@@ -0,0 +1,22 @@
+# Legal Notice
+
+_Last updated: {{entity.lastUpdated}}_
+
+## Publisher
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)